app/[domain]/page.tsx

```diff
@@ -200,6 +192,15 @@ export default async function IndexPage({
                     }
                   })()}
                 </p>
+              ) : (
+                <>
+                  {newHandle && (
+                    <p className="text-muted-forground mt-4 flex flex-row items-center gap-2 text-sm">
```

Comment:
The green tick is smaller than the one above, is it possible to make them the same size (larger)

![image](https://github.com/user-attachments/assets/92276a3c-1a30-4e9a-ae6e-b99aa44c639d)